posts/2021-04-16-mp-context-propagration-21004.adoc

```diff
@@ -0,0 +1,353 @@
+
+[source, xml]
+----
+krb5TicketCache="${server.config.dir}/security/krb5-user1.cc"
+----
+
+Set the Kerberos configuration file name (for example, krb5.conf or krb5.ini) using the Kerberos element:
+
+[source, xml]
+----
+<kerberos configFile="${server.config.dir}/security/krb5.conf"/>
+----
+
+If you are using a Kerberos keytab file, set the Kerberos keytab file name using the Kerberos element:
+
+[source, xml]
+----
+<kerberos keytab="${server.config.dir}/security/krb5.keytab" configFile="${server.config.dir}/security/krb5.conf"/>
+----
+
+If the Kerberos configuration file is not defined in the Kerberos element, Open Liberty will attempt to resolve the configuration file location by using the JDK default locations.
+
+For the Kerberos credentials, the locations are checked in the following order: the ticket cache (if provided), the configured keytab file, and finally the JDK default location.
+
+The following example shows how to configure the `LdapRegistry` element using a Kerberos ticket cache and Kerberos config file:
+
+[source, xml]
+----
+<kerberos configFile="${server.config.dir}/security/krb5.conf"/>
+
+<ldapRegistry id="LDAP" realm="SampleLdapADRealm" host="ldap_hostname" port="389" ignoreCase="true" baseDN="DC=example,DC=com" bindAuthMechanism="GSSAPI" krb5Principal="user1@EXAMPLE.COM" krb5TicketCache="${server.config.dir}/security/krb5-user1.cc" ldapType="Custom" />
+----
+
+The following example shows how to configure an LDAP Registry using a Kerberos keytab and Kerberos config file:
+
+[source, xml]
+----
+<kerberos keytab="${server.config.dir}/security/krb5.keytab" configFile="${server.config.dir}/security/krb5.conf" />
+
+<ldapRegistry id="LDAP" realm="SampleLdapADRealm" host="ldap_hostname" port="389" ignoreCase="true" baseDN="DC=example,DC=com" bindAuthMechanism="GSSAPI" krb5Principal="user1@EXAMPLE.COM" ldapType="Custom" />
+----
+
+For more information on `LdapRegistry`, see the link:https://openliberty.io/docs/latest/reference/feature/ldapRegistry-3.0.html[LDAP User Registry documentation].
+
+To enable this new beta function in your app, pull the All Beta Features package and add the LDAP User Registry 3.0 feature to your `server.xml` file:
+
+[source, xml]
+----
+<featureManager>
+  <feature>ldapRegistry-3.0</feature>
+</featureManager>
+----
+
+[#expandLocation]
+=== expandLocation property for AutoExpand
+
+With this enhancement users are now able to specify a expansion location (`expandLocation`) on the `applicationManager` configuration to be utilized when the `autoExpand` attribute is set to "true". As currently implemented, when an application is autoExpanded the default location for the expanded files are hard coded to `${server.config.dir}/apps/expanded/`.
+
+Now with this enhancement in place, the users are able to configure that location to a new value on the `filesystem`. Thus for example:
```

Comment:
Now with this enhancement in place, you can configure that location to a new value on the `filesystem`.  
For example, the following configuration snippet would cause the application to be expanded at `${server.config.dir}/myApps/{appname}/`:

```
[source, xml]
----
<applicationManager autoExpand="true" expandLocation="${server.config.dir}/myApps/" />
----
```

Reply:
note you'll need to handle the styling as I needed to add code block so that github renders the above

Reply:
Done